src/statement/hydrateResponse.ts

```diff
@@ -22,6 +27,13 @@ const getHydratedValue = (
     }
     return value;
   }
+  if (isByteAType(type)) {
+    if (!(value as string).startsWith(BYTE_A_PREFIX)) {
```

Comment:
not sure we should throw here. Packdb should be responsible for correct saved data

Reply:
I actually agree - I was concerned that users might be confused if they got a wrong response from the server - but yep it actually should never happen and it's not the responsibility of the connectors to deal with that.